src/libraries/Directory.Build.targets

```diff
@@ -295,4 +295,23 @@
     <DefaultValidateFramework Remove="net6.0" />
   </ItemGroup>
 
+  <PropertyGroup Condition="Exists('$(PkgProjPath)')"> 
+    <ServicingVersion Condition="'$(ServicingVersion)' == ''">0</ServicingVersion>
+    <!-- Always update the package version.-->
+    <PackageVersion>$(MajorVersion).$(MinorVersion).$(ServicingVersion)</PackageVersion>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="Exists('$(PkgProjPath)') and '$(AssemblyVersion)' == '$(MajorVersion).$(MinorVersion).0.0'">       
```

Comment:
Since you are only applying this when AssemblyVersion ends with 0.0, doesn't that mean that projects with hardcoded `AssemblyVersion` will not rev their 'netfx' version any more?